features/print.yml

```
@@ -0,0 +1,8 @@
+name: Printing
```

With a name this broad, how do we explain that CSS page breaking features aren't a part of it?

I think a group for printing might make more sense, and that we have one feature for `print()`, and another for the events. I assume the events also fire when the user initiates printing, so they're not tied to the API.

---

Since we're following caniuse here, we could do this (and rename the feature to something like `print-events`):

```suggestion
name: Printing events
```

I think this is defensible since the spec specifies that `beforeprint` and `afterprint` must fire for before and after `window.print()` as well as when the user requests printing.

---

`print` + `print-events` + `media-print` features sounds good to me. The last one not in this PR however, we'll need to think more about media query features in general.